ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/chatbot/clientlibs/js/inject.js

```diff
@@ -0,0 +1,32 @@
+const injectDOMElement = (tagName, targetSelector, options = {}) => {
+  const element = document.createElement(tagName)
+  Object.keys(options).forEach(key => (element[key] = options[key]))
+  document.querySelector(targetSelector).appendChild(element)
+  return element
+}
+
+window.addEventListener('message', ({ data }) => {
+  if (!data || !data.type || data.type !== 'setClass') {
+    return
+  }
+  document.querySelector('#bp-widget').setAttribute('class', data.value)
+})
+
+const init = config => {
+  const { host = '', botId = '' } = config
+  const cssHref = `${host}/assets/modules/channel-web/inject.css`
```

Comment:
@ReenaOmana : from where is the inject.css getting loaded ?

Reply:
inject.css is getting loaded from botpress. We tried replacing the file in our local, but it's not working. So we have overridden the file in chatbot.less